### ISSUE TITLE
It's a little difficult to understand the code（line 95-97）in HighResolutionModule.py

### ISSUE THREAD

**Issue by kubl88** [AI-Chen/HRNet-V2#2]
for k in range(len(temp_layer)):
                fuse = 0
                fuse = fuse + temp_layer[k]

Please have a look.
Thanks

**Comment by minogame** [comment]
The implementation of the `_make_fuse_layers` is totally incorrect, please check the official repo.

**Comment by Linfeng-Tang** [comment]
I don't think it handles downsampling well, and according to the official REPORT method, it should downsample i-j times when j<i (i.e. perform i-j conv(3×3, stride=2)), but in its implementation it only performs once
